update-d2c-service/src/core.ts

```diff
@@ -1,8 +1,11 @@
 import { createD2cApiWithAuth } from './d2c-api/create-d2c-api-with-auth';
 import { validateActions } from './utils';
+import * as YAML from 'yaml';
+import fs from 'fs';
 
 type UpdateServiceParams = {
```

Comment:
add config-path to action.yaml file.
update readme to have config-path variable.
make servieName optional in types and in action.yaml and in readme :)